cmd/mo-service/main.go

```diff
@@ -309,7 +309,7 @@ func initTraceMetric(ctx context.Context, st metadata.ServiceType, cfg *Config,
 	}
 
 	if !SV.DisableTrace || !SV.DisableMetric {
-		writerFactory = export.GetWriterFactory(fs, UUID, nodeRole, SV.LogsExtension)
+		writerFactory = export.GetWriterFactory(fs, UUID, nodeRole, table.CsvExtension)
```

Comment:
this should not be hard coded. should be control by config item as `SV.LogsExtension`
you could change default value to `csv`

Reply:
Will revert this after finishing the coding.